lib/ejson/rails/railtie.rb

```diff
@@ -20,6 +20,13 @@ class Railtie < Rails::Railtie
         Rails.application.credentials.config.deep_merge!(secrets) do |key|
           raise "A credential already exists with the same name: #{key}"
         end
+
+        # Delete the loaded JSON files so they are no longer readable by the app.
+        if ENV["EJSON_RAILS_DELETE_SECRETS"] == "true"
+          json_files.each do |pathname|
+            File.delete(pathname) if File.writable?(pathname)
```

Comment:
These are pathanmes, so we could've just used the object's API instead of passing it to File: https://docs.ruby-lang.org/en/3.2/Pathname.html#method-i-delete